Client/core/CMainMenu.cpp

```diff
@@ -221,6 +224,53 @@ CMainMenu::CMainMenu(CGUI* pManager)
     m_pLatestNews->SetProperty("InheritsAlpha", "False");
     m_pLatestNews->SetVisible(false);
 
+    static const std::array<std::pair<const char*, const char*>, 6> m_socialMediaLinks{{
```

Comment:
Move this out of `CMainMenu::CMainMenu` into file scope and remove the `m_` prefix, please.

Reply:
fixed